assets/js/components/setup/compatibility-checks.js

```diff
@@ -142,9 +165,44 @@ export default class CompatibilityChecks extends Component {
 						external={ external }
 						inherit
 					/>
-				</Fragment>;
+				</p>;
 			case ERROR_TOKEN_MISMATCH:
-				return __( 'Looks like you may be using a caching plugin which could interfere with setup. Please deactivate any caching plugins before setting up Site Kit. You may reactivate them once setup has been completed.', 'google-site-kit' );
+				return <p>{ __( 'Looks like you may be using a caching plugin which could interfere with setup. Please deactivate any caching plugins before setting up Site Kit. You may reactivate them once setup has been completed.', 'google-site-kit' ) }</p>;
+			case ERROR_GOOGLE_API_CONNECTION_FAIL:
+				return <Fragment>
+					<p
+						dangerouslySetInnerHTML={ sanitizeHTML(
+							sprintf(
+							/* translators: %s: Support Forum URL */
+								__( 'Looks like your site is having a technical issue with requesting data from Google services.<br/>To get more help, ask a question on our <a href="%1$s">support forum</a> and include the text of the original error message:<br/>%2$s', 'google-site-kit' ),
+								'https://wordpress.org/support/plugin/google-site-kit/',
+								error
+							),
+							{
+								ALLOWED_TAGS: [ 'a', 'br' ],
+								ALLOWED_ATTR: [ 'href' ],
+							}
+						) }
+
+					/>
+				</Fragment>;
+			case ERROR_AMP_CDN_RESTRICTED:
+				return <Fragment>
+					<p
+						dangerouslySetInnerHTML={ sanitizeHTML(
+							sprintf(
+							/* translators: %s: Support Forum URL */
```

Comment:
The translators comment needs to be updated to match the changes to the translated string (alternate placeholders and added placeholder).